### ISSUE TITLE
Issue with dof numbering in CDLagrangianRefFE

### ISSUE THREAD

**Issue by fverdugo** [gridap/Gridap.jl#302]
@santiagobadia I have realized that there is a problem with the dof numbering in `CDLagrangianRefFE`

The problem is that you have assumed lexicographical ordering of the high order nodes, but we are not adopting this numbering in Gridap at the moment.

Consider this example, where I have manually fixed the numeration so that we generate the FE Space we want:
```julia
using Gridap.ReferenceFEs
using Gridap
using Gridap.FESpaces

reffe = CDLagrangianRefFE(Float64,QUAD,(2,2),(CONT,DISC))
# Manual fix
reffe.face_own_dofs[7] = [1,3,7]
reffe.face_own_dofs[8] = [2,4,8]
reffe.face_own_dofs[9] = [5,6,9]

model = CartesianDiscreteModel((0,1,0,1),(10,5))

V = FESpace(model=model,reffe=reffe,conformity=true)

trian = Triangulation(model)

vh = FEFunction(V,rand(num_free_dofs(V)))

writevtk(trian,"trian",nsubcells=20,cellfields=["vh"=>vh])
```

![cdspace](https://user-images.githubusercontent.com/48508177/86002197-51a17400-ba10-11ea-93c7-955cefa830ff.png)

**Comment by fverdugo** [comment]
This is the numeration we adopt now for 2nd order Q elems:

```julia
reffe = LagrangianRefFE(Float64,QUAD,2)
writevtk(reffe,"reffe")
```

![dofs_QUAD9](https://user-images.githubusercontent.com/48508177/86002691-fae86a00-ba10-11ea-8ad2-85ef8539ff7c.png)

**Comment by santiagobadia** [comment]
Fixed in PR [#305](https://github.com/gridap/Gridap.jl/pull/305)
